docs/tutorials/walkthrough/odometry.md

```diff
@@ -0,0 +1,163 @@
+# Odometry
+
+Odometry is the act of tracking a robot's absolute position. 
+This information can be used by motion algorithms to drive to positions or turn to absolute angles.
+
+An excellent overview of the odometry tracking algorithm can be found [here](https://www.vexforum.com/t/team-5225-introduction-to-position-tracking-document/49640).
+
+# Configuring Odometry
+
+OkapiLib supports odometry for all chassis configurations, and is configured using 
```

Comment:
Can you rephrase this? It sounds awkward.

Reply:
```suggestion
OkapiLib supports odometry for all chassis configurations. A chassis controller that uses odometry is configured using the 
```
Better?